user_docs/en/userGuide.t2t

```diff
@@ -2589,33 +2589,67 @@ To toggle the braille viewer from anywhere, please assign a custom gesture using
 The NVDA Python console, found under Tools in the NVDA menu, is a development tool which is useful for debugging, general inspection of NVDA internals or inspection of the accessibility hierarchy of an application.
 For more information, please see the [NVDA Developer Guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html].
 
-++ Add-ons Manager ++[AddonsManager]
-The Add-ons Manager, accessed by selecting Manage add-ons under Tools in the NVDA menu, allows you to install, uninstall, enable and disable add-on packages for NVDA.
+++ Add-on Store ++[AddonsManager]
+The Add-on Store, accessed by the Tools submenu in the NVDA menu, allows you to browse and manage add-on packages for NVDA.
 These packages are provided by the community and contain custom code that may add or change features in NVDA or even provide support for extra Braille displays or speech synthesizers.
 
-The Add-ons Manager contains a list that displays all the add-ons currently installed in your NVDA user configuration. 
-Package name, status, version and author are shown for each add-on, though further information such as a description and URL can be viewed by selecting the add-on and pressing the About add-on button.
-If there is help available for the selected add-on, you can access it by pressing the Add-on help button.
+When exiting the Add-on Store, you may need to restart NVDA so that your changes can take effect.

```

Comment:
this statement looks a little out of context. Perhaps remove it from here, and place it in any subsection where this is true, such as installing, enabling or disabling.

Reply:
I agree, the purpose of the text directly under the main Add-on Store heading is to tell you what it is and how to get to it.  How it behaves on exit once you've performed certain tasks can be covered as those tasks are covered - especially as we are talking about restarting NVDA here.  If you were starting a process which may lead to needing to restart the whole PC, then sure, you'd want to know about that possibility before you go in, but restarting NVDA itself shouldn't have any adverse effects.

I would also be inclined to seperate the "how you get into it" from the "what it is" sentence, and pair it with the custom gesture comment, like:

"The Add-on Store allows you to browse and manage add-on packages for NVDA. These packages are provided by the community and contain custom code that may add or change features in NVDA or even provide support for extra Braille displays or speech synthesizers.

The add-on store is accessed by the Tools submenu in the NVDA menu.  To access the Add-on Store from anywhere, please assign a custom gesture using the Input Gestures dialog."

Reply:
>this statement looks a little out of context. Perhaps remove it from here, and place it in any subsection where this is true, such as installing, enabling or disabling.

I think we already cover the need to restart in these sections. I included it here as well for general posterity, however I will remove it and make the changes @Qchristensen suggested.